README.md

```diff
@@ -11,55 +11,91 @@ OpenColorIO
 
 Introduction
 ------------
+
 [linear]: https://origin-flash.sonypictures.com/ist/imageworks/cloudy1.jpg
 [log]: https://origin-flash.sonypictures.com/ist/imageworks/cloudy3.jpg
 [vd]: https://origin-flash.sonypictures.com/ist/imageworks/cloudy2.jpg
 
 ![lnh][linear] ![lm10][log] ![vd8][vd]
 
-OpenColorIO (OCIO) is a complete color management solution geared towards motion
-picture production with an emphasis on visual effects and computer animation.
-OCIO provides a straightforward and consistent user experience across all
-supporting applications while allowing for sophisticated back-end configuration
-options suitable for high-end production usage. OCIO is compatible with the
-Academy Color Encoding Specification (ACES) and is LUT-format agnostic,
-supporting many popular formats.
+OpenColorIO (OCIO) is a complete color management solution geared towards
+motion picture production with an emphasis on visual effects and computer
+animation. OCIO provides a straightforward and consistent user experience
+across all supporting applications while allowing for sophisticated back-end
+configuration options suitable for high-end production usage. OCIO is
+compatible with the Academy Color Encoding Specification (ACES) and is
+LUT-format agnostic, supporting many popular formats.
 
 OpenColorIO is released as version 1.0 and has been in development since 2003.
-OCIO represents the culmination of years of production experience earned on such
-films as SpiderMan 2 (2004), Surf's Up (2007), Cloudy with a Chance of Meatballs
-(2009), Alice in Wonderland (2010), and many more. OpenColorIO is natively
-supported in commercial applications like Katana, Mari, Nuke, Silhouette FX, and others.
+OCIO represents the culmination of years of production experience earned on
+such films as SpiderMan 2 (2004), Surf's Up (2007), Cloudy with a Chance of
+Meatballs (2009), Alice in Wonderland (2010), and many more. OpenColorIO is
+natively supported in commercial applications like Katana, Mari, Nuke,
+Silhouette FX, and others.
```

Comment:
It might be worth mentioning Maya here too since it is one of the other big apps in the VFX world.

Reply:
That crossed my mind while revising this. Completely agree. I'll add it.